### ISSUE TITLE
PHPLint 2 Compatibility

### ISSUE THREAD

**Issue by Hephistocles** [SublimeLinter/SublimeLinter-phplint#8]
Earlier this year, PHPLint 2 was released. When I tried to use this plugin with PHPLint 2, no errors were shown in the Sublime console, and linting results were correctly output to this console, but no highlighting was shown in my file. I don't know if the PHPLint rewrite has had breaking changes to the output or something!

Below is the console output indebug mode:
```
SublimeLinter: phplint: test.php ['C:\\php\\phplint.bat', '--print-path', 'shortest', '--print-context', '--tab-size', '4', '--no-overall'] 
SublimeLinter: phplint output:
BEGIN parsing of ../../christ~1/appdata/local/temp/SublimeLinter3/test.php
1:	<?php
2:		ERROR ;

		ERROR ;
	           \_ HERE
==== 2: ERROR: unknown constant ERROR
3:	?>
END parsing of ../../christ~1/appdata/local/temp/SublimeLinter3/test.php
Total execution time: 176 ms 
```

**Comment by SavageCore** [comment]
Decided to fork and create the pull request to implement @bwbroersma changes. Hope that is ok!

**Comment by grotewold** [comment]
merged in with #11
